src/Core/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -1343,7 +1352,7 @@ private async Task PopulateForeignKeyDefinitionAsync()
 
             // Gather all the referencing and referenced columns for each pair
             // of referencing and referenced tables.
-            PairToFkDefinition = await QueryExecutor.ExecuteQueryAsync(queryForForeignKeyInfo, parameters, SummarizeFkMetadata);
+            PairToFkDefinition = await QueryExecutor.ExecuteQueryAsync(queryForForeignKeyInfo, parameters, SummarizeFkMetadata, null, null, _dataSourceName);
```

Comment:
nit: name the parameters that are `null`

Reply:
fixed.